autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -1,7 +1,11 @@
-# Sets up the environment for autobuild on Windows
+# Powershell
```

Comment:
Are you sure this comment is better than the one from before?

Reply:
line 3 is "# autobuild_1_prepare: set up environment, install Qt & dependencies" << thats the one to compare, and I think it is more concrete.

line 1 mimmics the shebang of other files, so mac/linux/windows scripts have the same header format

Reply:
Hmm. Ok. Will resolve this too.